internal/factory/container/container.go

```diff
@@ -215,11 +215,11 @@ func (c *container) SpecAddAnnotations(ctx context.Context, sb *sandbox.Sandbox,
 	}
 
 	c.spec.AddAnnotation(annotations.UserRequestedImage, userRequestedImage)
-	imageName := ""
+	someNameOfThisImage := ""
```

Comment:
I'm not personally a huge fan of how verbose this variable is. What is the risk to cri-o programmers if we leave the name as-is? do the comments present not document that risk?

Reply:
Yes, I know you’re not a fan. I’ll take it out if you decide so.

The risk has materialized in https://github.com/cri-o/cri-o/pull/8212#discussion_r1734986296 / https://github.com/cri-o/cri-o/pull/8212#discussion_r1747337067 . (And I don’t know what https://github.com/cri-o/cri-o/pull/8590#discussion_r1752967914 deos.)

I don’t know that these ugly verbose names would _help_ but also I don’t have any better ideas.

---

Ideally, perhaps, CRI-O would change the logic to record some other values that are more known to be relevant (e.g. `PullImage` now returns a pretty well-specified repo@digest value). But that’s not this PR, and it would almost certainly change the API outputs; so I’ll leave the CRI-O maintainers to navigate that.

The ugly verbose names might serve as a gentle encouragement :)

Reply:
Further discussion in https://github.com/cri-o/cri-o/pull/8706#discussion_r1813429259 , I have dropped this part from this PR.